package.json

```diff
@@ -78,7 +78,10 @@
     "set-value": "^2.0.1",
     "js-yaml": "^3.13.1",
     "acorn": "^6.4.1",
-    "minimist": "^1.2.2"
+    "minimist": "^1.2.2",
+    "serialize-javascript": "^3.1.0",
```

Comment:
Yarn suggests putting resolutions inside package.json, best I can tell. See https://classic.yarnpkg.com/en/docs/selective-version-resolutions/

Reply:
I see, we can do that since it's the recommendation from yarn.

Reply:
I'm going to try and update the upstream packages `yarn why` tells me these come from, to see if that'll also help clear it up. Webpack is one, for example. But I wanted to try this first. It may be some of these get removed depending on how that goes.